src/Microsoft/Silk.NET.DirectComposition/Structs/TagCOMPOSITIONSTATS.gen.cs

```diff
@@ -0,0 +1,72 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+
+using System;
+using System.Runtime.InteropServices;
+using System.Runtime.CompilerServices;
+using System.Text;
+using Silk.NET.Core;
+using Silk.NET.Core.Native;
+using Silk.NET.Core.Attributes;
+using Silk.NET.Core.Contexts;
+using Silk.NET.Core.Loader;
+
+#pragma warning disable 1591
+
+namespace Silk.NET.DirectComposition
+{
+    [NativeName("Name", "tagCOMPOSITION_STATS")]
+    public unsafe partial struct TagCOMPOSITIONSTATS
```

Comment:
Here too, the name of the struct is wrong, i think something in our rename logic is bad, or theres a rouge typemap

Reply:
I personally would manually rename this one as I haven't added anything to remove the `tag` prefix, so I've manually renamed structs like this elsewhere anyway.